tests/queries/0_stateless/03224_arrayUnion.sql

```diff
@@ -0,0 +1,50 @@
+drop table if exists array_union;
+
+create table array_union (date Date, arr Array(UInt8)) engine=MergeTree partition by date order by date;
+
+insert into array_union values ('2019-01-01', [1,2,3]);
+insert into array_union values ('2019-01-01', [1,2]);
+insert into array_union values ('2019-01-01', [1]);
+insert into array_union values ('2019-01-01', []);
+
+
+select arraySort(arrayUnion(arr, [1,2])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion(arr, [])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([], arr)) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([1,2], arr)) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([1,2], [1,2,3,4])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([], [])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion(arr, [1,2])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion(arr, [])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([], arr)) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([1,2], arr)) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([1,2], [1,2,3,4])) from array_union order by arr;
+select '-------';
+select arraySort(arrayUnion([], [])) from array_union order by arr;
```

Comment:
Seems that these lines are the same with line 11 to line 21.

Reply:
Yep thanks for catching that